DatabaseMySQL/Dump20221018.sql

```diff
@@ -0,0 +1,200 @@
+  `email` VARCHAR(50) NOT NULL,
+  `password_hash` VARCHAR(50) NOT NULL,
+  `is_host` BIT(1) NOT NULL DEFAULT 0,
+  PRIMARY KEY (`user_id`),
+  UNIQUE INDEX `email_UNIQUE` (`email` ASC) VISIBLE)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `dbQuiz`.`subjects`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `dbQuiz`.`subjects` (
+  `subject_id` BIGINT(20) NOT NULL,
+  `subject_name` VARCHAR(50) NOT NULL,
+  PRIMARY KEY (`subject_id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `dbQuiz`.`exams`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `dbQuiz`.`exams` (
+  `exam_id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+  `subject_id` BIGINT(20) NOT NULL,
+  `title` VARCHAR(50) NOT NULL,
+  `total_question` INT(5) NOT NULL DEFAULT 0,
+  `total_score` INT(5) NOT NULL DEFAULT 0,
+  `score_per_question` DOUBLE NOT NULL DEFAULT 0.0,
+  PRIMARY KEY (`exam_id`),
+  INDEX `subjectid_fk_idx` (`subject_id` ASC) VISIBLE,
+  CONSTRAINT `subject_fk`
+    FOREIGN KEY (`subject_id`)
+    REFERENCES `dbQuiz`.`subjects` (`subject_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `dbQuiz`.`rooms`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `dbQuiz`.`rooms` (
+  `room_id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+  `exam_id` BIGINT(20) NOT NULL,
+  `password` VARCHAR(50) NULL DEFAULT NULL,
+  `title` VARCHAR(50) NOT NULL,
+  `time_limit` INT(5) NOT NULL DEFAULT 0,
+  `is_available` BIT(1) NOT NULL DEFAULT 0,
+  PRIMARY KEY (`room_id`),
+  INDEX `examid_fk_idx` (`exam_id` ASC) VISIBLE,
+  CONSTRAINT `examid_fk`
+    FOREIGN KEY (`exam_id`)
+    REFERENCES `dbQuiz`.`exams` (`exam_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `dbQuiz`.`questions`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `dbQuiz`.`questions` (
+  `question_id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+  `subject_id` BIGINT(20) NOT NULL,
+  `level` INT(5) NOT NULL DEFAULT 1,
+  `content` TEXT NULL DEFAULT NULL,
+  PRIMARY KEY (`question_id`),
+  INDEX `subjectid_fk_idx` (`subject_id` ASC) VISIBLE,
+  CONSTRAINT `subjectid_fk`
+    FOREIGN KEY (`subject_id`)
+    REFERENCES `dbQuiz`.`subjects` (`subject_id`)
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `dbQuiz`.`question_answers`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `dbQuiz`.`question_answers` (
+  `question_answer_id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+  `question_id` BIGINT(20) NOT NULL,
+  `is_correct` BIT(1) NOT NULL DEFAULT 0,
+  `content` TEXT NULL DEFAULT NULL,
```

Comment:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/66414511/196325425-49fe68ef-b387-4758-9564-ad9d42bfd5f5.png">

Reply:
đáp án cho câu hỏi là phải có dữ liệu nhé, nên ko NOT NULL đc